### DISCUSSION TITLE
How to do a POST api gateway with input via POST body ?

### DISCUSSION THREAD

**Discussion by marifrahman** [apioo/fusio#536] [Q&A]
I want to do a api gateway for a POST method where an input is given via body. I can see the GET is working but not sure how to do the same for a POST. Any help will be much appreciated.
Example I can see is here : https://www.youtube.com/watch?v=S-mneuDPNzg

The api I am trying looks in postman as bellow : 
<img width="646" alt="image" src="https://github.com/apioo/fusio/assets/772407/89df99a8-ede0-40c5-9785-1f9b2b6d5f0d">

**Comment by chriskapp** [comment]
Hi @marifrahman so if you use the HTTP-Processor action the body should be forwarded to the configured endpoint. In your case maybe the operation is only configured for the GET method, you would need to create a separate operation for the POST method so that you action gets invoked on POST.

**Reply by marifrahman** [reply]
Thanks again @chriskapp. 
My question is how you pass from an Operation to Action ? This is a Operation I am creating :
<img width="593" alt="image" src="https://github.com/apioo/fusio/assets/772407/ebddf82d-9dd1-433d-a6b3-04a81d68b46e">

And the action used looks like this : 
<img width="594" alt="image" src="https://github.com/apioo/fusio/assets/772407/d3bffd5f-914f-4969-a75c-b2d8a8068b55">

**Reply by marifrahman** [reply]
I think I figured it out :) My operation should look like this to passThrough the body. My bad but thanks a lot : 
<img width="590" alt="image" src="https://github.com/apioo/fusio/assets/772407/072c0c87-269f-4755-930c-75324dba513d">
